core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigCompatibility.java

```diff
@@ -55,12 +55,12 @@ public final class ConfigCompatibility {
                     ConfigCompatibility::quarkusPackageIncludedOptionalDependencies),
             entry(List.of("quarkus", "package", "include-dependency-list"),
                     ConfigCompatibility::quarkusPackageIncludeDependencyList),
-            entry(List.of("quarkus", "package", "vineflower", "version"),
-                    ConfigCompatibility::quarkusPackageVineflowerVersion),
-            entry(List.of("quarkus", "package", "vineflower", "enabled"),
-                    ConfigCompatibility::quarkusPackageVineflowerEnabled),
-            entry(List.of("quarkus", "package", "vineflower", "jar-directory"),
-                    ConfigCompatibility::quarkusPackageVineflowerJarDirectory),
```

Comment:
Don't we need to keep these entries and redirect them too?

Reply:
Yes *however* there is some history here - see this series of changes (in reverse chronological order, but the first one is the important one):

- #37445
- #35610
- #25729 / #30909 / #33673

How far should I go to ensure compatibility? I can do any of:

- Just map old Vineflower config
- Map old Vineflower and Quiltflower configs
- Map all of Vineflower, Quiltflower, and Fernflower configs

Reply:
I went with just mapping the `vineflower` ones and leaving it at that.